src/main/fc/rc.c

```diff
@@ -335,77 +335,72 @@ bool getRxRateValid(void)
 
 #ifdef USE_RC_SMOOTHING_FILTER
 
+static FAST_CODE_NOINLINE void rcSmoothingUpdateFilterTau(rcSmoothingFilter_t *smoothingData)
+{
+    const float cen_tau = smoothingData->setpointTauCenter;
+    const float end_tau = smoothingData->setpointTauEnd;
+
+    const float dT = targetPidLooptime * 1e-6f;
+
+    for (int i = FD_ROLL; i <= FD_YAW; i++) {
+        const float tau = (1.0f - rcDeflectionAbs[i]) * cen_tau + rcDeflectionAbs[i] * end_tau;
```

Comment:
(maybe it is time to improve math.h: (GPT3o suggest that `lepr` is de facto standard) 
```
static inline float lerp(float t, float a, float b) {
    return a + t * (b - a);
}
```